Tests/scripts/validate.sh

```diff
@@ -1,16 +1,22 @@
 #!/usr/bin/env bash
 set -ex
 
-echo "CI_COMMIT_BRANCH: $CI_COMMIT_BRANCH CI: $CI DEMISTO_README_VALIDATION: $DEMISTO_README_VALIDATION"
+
+echo "CI_COMMIT_BRANCH: $CI_COMMIT_BRANCH CI: $CI DEMISTO_README_VALIDATION: $DEMISTO_README_VALIDATION, CI_COMMIT_SHA: $CI_COMMIT_SHA, LAST_UPLOAD_COMMIT: $LAST_UPLOAD_COMMIT"
 if [[ $CI_COMMIT_BRANCH = master ]] || [[ -n "${NIGHTLY}" ]] || [[ -n "${BUCKET_UPLOAD}" ]] || [[ -n "${DEMISTO_SDK_NIGHTLY}" ]]; then
     if [[ -n "${PACKS_TO_UPLOAD}" ]]; then
         echo "Packs upload - Validating only the supplied packs"
         PACKS_TO_UPLOAD_SPACED=${PACKS_TO_UPLOAD//,/ }
         for item in $PACKS_TO_UPLOAD_SPACED; do
             python3 -m demisto_sdk validate -i Packs/"$item" --post-commit --graph --skip-pack-dependencies
         done       
-    else 
-        python3 -m demisto_sdk validate -a --post-commit --graph --skip-pack-dependencies
+    else
+        if [ -n "$NIGHTLY" ]; then
+            PREV_VER=$LAST_UPLOAD_COMMIT
+        else
+            PREV_VER="origin/master"
```

Comment:
Instead of adding this variable here, you can add it as a variable to the gitlab.yml file. There is an option to define different variables under different conditions(rules). For example: https://docs.gitlab.com/ee/ci/yaml/#rulesvariables
We already using different variables under different rules, look in the on-push file :).

Reply:
defined https://github.com/demisto/content/blob/master/.gitlab/ci/.gitlab-ci.global.yml#L184,
is it considered to be defined globally? i am not sure.

Reply:
Instead of defined globally, I should write - predefined when the build starts. 
Sorry for that. 
It is ok you can implement my suggestion because of the way it is defined. 
You can ignore this comment. thanks.